### ISSUE TITLE
[live_demo] All tree content is ellipses(ed/sid/ish?)

### ISSUE THREAD

**Issue by larkin** [nasa/openmct#887]
This makes it a bit hard to follow the steps in the demo and it's not clear how to resize the tree.  Can we make the tree just a little bit wider by default?

![screen shot 2016-05-05 at 3 57 49 pm](https://cloud.githubusercontent.com/assets/226503/15059804/261eb5d6-12da-11e6-8d17-7b2ce4fdf03a.png)

**Comment by charlesh88** [comment]
Agree. I had the exact same thought and was going to file an issue myself for this.
